Firmware/Marlin_main.cpp

```diff
@@ -2508,23 +2503,19 @@ void gcode_M105(uint8_t extruder)
         SERIAL_PROTOCOLPGM(" Rxb->");
         SERIAL_PROTOCOL_F(raw, 5);
 #endif
-        for (int8_t cur_extruder = 0; cur_extruder < EXTRUDERS; ++cur_extruder) {
-            SERIAL_PROTOCOLPGM("  T");
-            SERIAL_PROTOCOL(cur_extruder);
-            SERIAL_PROTOCOLPGM(":");
-            SERIAL_PROTOCOL_F(degHotend(cur_extruder),1);
-            SERIAL_PROTOCOLPGM("C->");
-            raw = rawHotendTemp(cur_extruder);
-            SERIAL_PROTOCOL_F(raw/OVERSAMPLENR,5);
-            SERIAL_PROTOCOLPGM(" Rt");
-            SERIAL_PROTOCOL(cur_extruder);
-            SERIAL_PROTOCOLPGM("->");
-            SERIAL_PROTOCOL_F(100 * (1 + (PtA * (raw/OVERSAMPLENR)) + (PtB * sq((raw/OVERSAMPLENR)))), 5);
-            SERIAL_PROTOCOLPGM(" Rx");
-            SERIAL_PROTOCOL(cur_extruder);
-            SERIAL_PROTOCOLPGM("->");
-            SERIAL_PROTOCOL_F(raw, 5);
-        }
+        SERIAL_PROTOCOLPGM("  T0:");
+        SERIAL_PROTOCOL_F(degHotend(active_extruder),1);
+        SERIAL_PROTOCOLPGM("C->");
+        raw = rawHotendTemp(active_extruder);
+        SERIAL_PROTOCOL_F(raw/OVERSAMPLENR,5);
+        SERIAL_PROTOCOLPGM(" Rt");
+        SERIAL_PROTOCOL(active_extruder);
```

Comment:
now, since we don't support >1 extruders anymore, we could merge these serial_protocol calls into a single string:
```
SERIAL_PROTOCOLPGM(" Rt");
SERIAL_PROTOCOL(active_extruder);
SERIAL_PROTOCOLPGM("->");
```
into
```
SERIAL_PROTOCOLPGM(" Rt1->");
```
the same principle may apply elsewhere

Reply:
Fixed in [b9847ec](https://github.com/prusa3d/Prusa-Firmware/pull/3903/commits/b9847ec34303f6d9a7efa33e0aeed561f80b3fe2)